test/test_manifest.py

```diff
@@ -5,17 +5,16 @@
 
 import testutil
 
-
 if not testutil.has_executable("podman"):
     pytest.skip("no podman, skipping integration tests that required podman", allow_module_level=True)
 
 from containerbuild import build_container_fixture  # noqa: F401
 from testcases import gen_testcases
 
 
-@pytest.mark.parametrize("image_type", gen_testcases("manifest"))
-def test_manifest_smoke(build_container, image_type):
-    container_ref = image_type.split(",")[0]
+@pytest.mark.parametrize("images", gen_testcases("manifest"))
```

Comment:
(super nitpick) I don't think this is strictly needed, the original code is confusing, I will do a followup with a small cleanup.